zk_toolbox/crates/common/src/version.rs

```diff
@@ -0,0 +1,24 @@
+const GIT_VERSION: &str = git_version_macro::build_git_revision!();
+const GIT_BRANCH: &str = git_version_macro::build_git_branch!();
+const GIT_SUBMODULES: &[(&str, &str)] = git_version_macro::build_git_submodules!();
```

Comment:
@Deniallugo you can make this constant public later if you will want to check the local submodule version against compiled one (to avoid code duplication).